### DISCUSSION TITLE
Web Vitals

### DISCUSSION THREAD

**Discussion by rajeev** [LuXDAmore/nuxt-prune-html#31] [Q&A]
There is a statement in the README: "Faster web-vitals, faster TTI, faster FCP, faster FMP, faster all."

But this is not true. Google does not seem to care about Lab measurements of LCP, FDI, CLS. Field Data and Origin Summary are generated based on the LCP/FDI/CLS experienced by NON-bot users, i.e., non-pruned HTML (based on CrUX measurements)

Using prune-html obfuscates these measurements and makes it difficult to meet the latest Google CWV ranking updates. For people using prune-html for performance should keep this in mind.

Prune-html is not a silver bullet, and you have to be very careful when using it.

**Comment by LuXDAmore** [comment]
You're right @rajeev ! 😄 

There is no _silver bullet_ for lazy developers!


You always have to care about performance, **this is an extra**!
Also useful for having more crawled pages for example!

Everyone wants to have the fastest website, but I'm a just developer, not a miracle worker! 🐱
